Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml

```diff
@@ -1114,57 +1139,12 @@ tasks:
       {
         "position": {
           "x": 160,
-          "y": 4530
-        }
-      }
-    note: false
-    timertriggers: []
-    ignoreworker: false
-    skipunavailable: false
-    quietmode: 0
-    isoversize: false
-    isautoswitchedtoquietmode: false
-  "92":
-    id: "92"
-    taskid: e026a817-ada8-4a83-8ef1-2e26a98fd8c6
-    type: regular
-    task:
-      id: e026a817-ada8-4a83-8ef1-2e26a98fd8c6
-      version: -1
-      name: Set User Containment Executed
-      description: Set a value in context under the key you entered.
-      scriptName: Set
-      type: regular
-      iscommand: false
-      brand: ""
-    nexttasks:
-      '#none#':
-      - "82"
-    scriptarguments:
-      append:
-        simple: "true"
-      key:
-        simple: RDP_Brute_Force_Investigation.User_Containment
-      value:
-        simple: Executed
-    separatecontext: false
-    continueonerrortype: ""
-    view: |-
-      {
-        "position": {
-          "x": -630,
-          "y": 300
+          "y": 4840
         }
       }
     note: false
     timertriggers: []
     ignoreworker: false
-    fieldMapping:
-    - incidentfield: User Disabled Status
```

Comment:
don't u want to map the other field you created?

Reply:
I am, using mapping this time. search for "User Block Status"